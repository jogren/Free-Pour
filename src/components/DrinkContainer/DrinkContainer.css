.DrinkContainer_section {
  display: flex;
  flex-direction: column;
}