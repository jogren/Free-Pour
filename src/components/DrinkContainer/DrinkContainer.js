import React from 'react';
import Drink from '../Drink/Drink';
import './DrinkContainer.css';

const DrinkContainer = ({ drinks }) => {
  let drinkList = drinks.map((drink, index) => {
    return <Drink key={index} image={drink.strDrinkThumb} name={drink.strDrink} />
  })

  return(
    <section className="DrinkContainer_section">
      {drinkList}
    </section>
  )
}

export default DrinkContainer;