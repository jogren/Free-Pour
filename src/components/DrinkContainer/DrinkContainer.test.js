import React from 'react';
import { shallow } from 'enzyme';
import DrinkContainer from './DrinkContainer';

describe('DrinkContainer', () => {
  let wrapper;
  const mockCurrentCocktails = [
    {
      strDrink: 'Mojito',
      strDrinkThumb: 'image.png'
    },
    {
      strDrink: 'Moscow Mule',
      strDrinkThumb: 'image2.png'
    }
  ]
  beforeEach(() => {
    wrapper = shallow(<DrinkContainer 
      drinks={mockCurrentCocktails}
    />)
  })

  it('should take a snapshot when passed the correct data', () => {
    expect(wrapper).toMatchSnapshot();
  })
})