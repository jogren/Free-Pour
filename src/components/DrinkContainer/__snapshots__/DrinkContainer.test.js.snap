// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DrinkContainer should take a snapshot when passed the correct data 1`] = `
<section
  className="DrinkContainer_section"
>
  <Connect(Drink)
    image="image.png"
    key="0"
    name="Mojito"
  />
  <Connect(Drink)
    image="image2.png"
    key="1"
    name="Moscow Mule"
  />
</section>
`;
